license_test.go

```diff
@@ -194,3 +201,24 @@ func TestMockLicenses(t *testing.T) {
 func TestLicenses(t *testing.T) {
 	CheckLicenses(t)
 }
+
+func TestLicensesWithEnterprise(t *testing.T) {
+	// Should produce the same result as nothing.
+	SetFirstEnterpriseCommit("HEAD")
+	defer SetFirstEnterpriseCommit("")
+	CheckLicenses(t)
+}
+
+func TestCommercialLicense(t *testing.T) {
+	// Test a commercial license in a temporary folder.
+	tmpdir, err := ioutil.TempDir("", "")
+	require.NoError(t, err)
+	defer os.RemoveAll(tmpdir)
+
+	abspath, err := filepath.Abs("./test_commercial_license.sh")
+	require.NoError(t, err)
+	cmd := exec.Command(abspath, abspath)
+	cmd.Dir = tmpdir
+	err = cmd.Run()
+	require.NoError(t, err)
```

Comment:
Are you completely sure they should all be requires? Like shouldn't this one be an assert?

Reply:
Well spotted. Originally I thought I could save a whole vendor directory, since there was only one of it. But `require` internally depends on `assert` anyway, so there was nothing to save. I've changed it.